src/tokenize.c

```diff
@@ -84,6 +84,9 @@ uint32_t simpleTokenizer_Next(RSTokenizer *base, Token *t) {
     size_t normLen = origLen;
     if (normLen > MAX_NORMALIZE_SIZE) {
       normLen = MAX_NORMALIZE_SIZE;
+      //Omer 
+      // Add error handling
+      if (overflow) *overflow = true;
```

Comment:
Seems also setting to `false` should be done here rather than by the caller.

Reply:
Since this is a breaking change - we can consider increasing `MAX_NORMALIZE_SIZE`